composer.json

```diff
@@ -1,46 +1,63 @@
 {
-    "name": "league/oauth1-client",
-    "description": "OAuth 1.0 Client Library",
-    "license": "MIT",
-    "require": {
-        "php": ">=5.3.0",
-        "guzzle/guzzle": "3.*"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "~4.0",
-        "mockery/mockery": "~0.9",
-        "squizlabs/php_codesniffer": "~2.0"
-    },
-    "keywords": [
-        "oauth",
-        "oauth1",
-        "authorization",
-        "authentication",
-        "idp",
-        "identity",
-        "sso",
-        "single sign on",
-        "bitbucket",
-        "trello",
-        "tumblr",
-        "twitter"
-    ],
-    "authors": [
-        {
-            "name": "Ben Corlett",
-            "email": "bencorlett@me.com",
-            "homepage": "http://www.webcomm.com.au",
-            "role": "Developer"
-        }
-    ],
-    "autoload": {
-        "psr-4": {
-            "League\\OAuth1\\": "src/"
-        }
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.0-dev"
-        }
+  "name": "league/oauth1-client",
+  "description": "OAuth 1.0 Client Library",
+  "license": "MIT",
+  "require": {
+    "php": ">=5.5.0",
+    "guzzlehttp/guzzle": "^6.0",
+    "ircmaxell/random-lib": "^1.1"
+  },
+  "require-dev": {
+    "henrikbjorn/phpspec-code-coverage": "^2.0",
+    "phploc/phploc": "^3.0",
+    "phpspec/phpspec": "^2.5",
+    "satooshi/php-coveralls": "^1.0",
+    "sebastian/phpcpd": "^2.0",
+    "squizlabs/php_codesniffer": "^2.6"
+  },
+  "keywords": [
+    "oauth",
+    "oauth2",
+    "authorization",
+    "authentication",
+    "idp",
+    "identity",
+    "sso",
+    "single sign on"
+  ],
+  "authors": [
+    {
+      "name": "Ben Corlett",
+      "email": "bencorlett@me.com",
+      "homepage": "http://webcomm.io",
+      "role": "Developer"
     }
+  ],
+  "autoload": {
+    "psr-4": {
+      "League\\OAuth1\\Client\\": "src/"
+    }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.0.x-dev",
+      "dev-develop": "2.0.x-dev"
+    }
+  },
+  "scripts": {
+    "phpcs": "./vendor/bin/phpcs --standard=psr2 src",
+    "phpcpd": "./vendor/bin/phpcpd src",
+    "phploc-log": "./vendor/bin/phploc --log-csv build/loc.csv --progress --git-repository . src",
+    "phploc": "./vendor/bin/phploc src",
+    "phpspec": "./vendor/bin/phpspec run --no-interaction",
+    "behat": "./vendor/bin/behat --tags '@completed' --no-interaction",
+    "test": [
+      "@composer install",
+      "@phpcs",
+      "@phpcpd",
+      "@phploc",
```

Comment:
The underlying package here requires at least PHP 5.6 and this package currently supports 5.5. Should we skip inclusion of this package?